Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -20,6 +20,11 @@ then
 elif [[ -n "$HOMEBREW_LINUX" ]]
 then
   case "$HOMEBREW_PROCESSOR" in
+    arm*)
+      ruby_URL="https://homebrew.bintray.com/bottles-portable-ruby/portable-ruby-2.3.3_2.armv6_linux.bottle.tar.gz"
+      ruby_URL2="https://github.com/Homebrew/homebrew-portable-ruby/releases/download/2.3.3_2/portable-ruby-2.3.3.armv6_linux.bottle.tar.gz"
```

Comment:
This (and x86_64 below) are missing the trailing `_2` in the filename. Also, can you move this whole block below `x86_64` because it's less common.